src/test/java/com/salesforce/dynamodbv2/mt/mappers/CreateTableRequestBuilderTest.java

```diff
@@ -87,9 +87,9 @@ void testExpectedBillingMode(ProvisionedThroughput inputProvisionedThroughput, B
             expectedCapacityUnits = 1L;
         }
 
-        testBuilder.withBillingMode(expectedBillingMode);
-        testBuilder.addSi(indexName, indexType, primaryKey, expectedCapacityUnits);
-        testBuilder.build();
+        testBuilder.withBillingMode(expectedBillingMode)
+            .addSi(indexName, indexType, primaryKey, expectedCapacityUnits)
+            .build();
 
         DynamoDbTestUtils.assertExpectedBillingModeIsSet(
             testBuilder.getCreateTableRequest(), expectedBillingMode, expectedCapacityUnits);
```

Comment:
why is a method on `testBuilder` being called after `build()`?

Reply:
I think I fixed this in 7c582987ce28bd844ea9d40d909dc7552de400c0.  @degaydorus, can you confirm?

Reply:
`getCreateTableRequest()` is exposed for test purposes only. In the test, the instance of `CreateTableRequestBuilder` is `testBuilder`.  `build()` modifies the member variable `createTableRequest` for default options.  The test calls the getter to verify this.

Reply:
perhaps a better name for `testBuilder`?

Reply:
also Mike pointed out `build()` returns the createTableRequest variable, so we actually don't need the getter for testing

Reply:
I think the name `testBuilder` is fine.  I tend to use `SUT`, for system under test(http://xunitpatterns.com/SUT.html), which is a standard xunit patterns(http://xunitpatterns.com/) term, which I always liked but no one else seems to be aware of.

Reply:
Yep you fixed it, I had posted the message before seeing your fix